src/components/PlayerName/index.tsx

```diff
@@ -1,21 +1,49 @@
+import { FC, useRef } from 'react';
+import { useDrag, useDrop } from 'react-dnd';
 import { Player } from "@/types";
 import classNames from "classnames";
-import { FC } from "react";
+import {  usePlayersStore } from "@/store";
 
 interface PlayerNameProps {
   player: Player;
   className?: string;
 }
 
-const PlayerName: FC<PlayerNameProps> = ({ player, className }) => (
-  <p className={classNames("flex items-center gap-1", className)}>
-    {player.name}
-    {player.details && (
-      <span className="flex text-[9px] font-semibold uppercase">
-        (<span className="block truncate max-w-12">{player.details}</span>)
-      </span>
-    )}
-  </p>
-);
+const PlayerName: FC<PlayerNameProps> = ({ player, className }) => {
+  const { exchangePlayers } = usePlayersStore();
```

Comment:
Don't import from the store directly

```suggestion
  const { exchangePlayers } = usePlayers();
```

Reply:
What do you recommend doing? Previously it was imported from the hook but code climate asked me to refactor

Reply:
Code climate wasn't warning that; it just wanted you to refactor the usePlayers hook, and it's done now. 
But it would be best if you used usePlayers directly here, which is enough.